release-tools/changelog.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python3
+
+import argparse
+import datetime
+import os
+import re
+
+import debian.changelog
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(description="automatic changelog writer for snapd")
+    parser.add_argument("version", type=str, help="new snapd version")
+    parser.add_argument(
+        "changelog",
+        type=argparse.FileType("r"),
+        help="path to new changelog entry as generated by snappy-dch",
+    )
+    return parser.parse_args()
+
+
+debianishDistros = [
```

Comment:
(nitpick) python is still on pep8, i.e. "deibanish_distros" for functions and vars. It's probably fine to merge as is but if/when I touch this I may run this through pep8 :)

Reply:
sure, like I said, I'm not up to speed on pythonic things or pep8, etc. so happy to apply feedback

Reply:
I renamed all the things to use underscores